plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_exporter_to_intermediate_schema.py

```diff
@@ -1238,7 +1266,18 @@ def _export_bio_ecol_assessment(self):
     def _export_fountain(self):
         query = self.tww_session.query(self.model_classes_tww_od.fountain)
         if self.filtered:
-            query = query.filter(self.model_classes_tww_od.fountain.obj_id.in_(self.subset_ids))
+            # creates duplicate alias fountain / connection_object to wwn
+            # query = query.join(self.model_classes_tww_od.wastewater_networkelement).filter(
+            # self.model_classes_tww_od.wastewater_networkelement.obj_id.in_(self.subset_ids)
+            # )
+            query = query.filter(
+                self.model_classes_tww_od.wastewater_networkelement.obj_id.in_(self.subset_ids)
+            )
+        # for debugging
```

Comment:
For debugging the plugin or to debug import/export issues in an easy way?

Reply:
I need to know how this filter query looks like in postgres notation - so I am trying to write it out to be able to copy it and check whether the selection criteria is correct.